applications/nrf_desktop/description.rst

```diff
@@ -952,6 +952,21 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for information abou
 
 .. nrf_desktop_fastpair_important_end
 
+nRF54L MCUboot provisioning
+===========================
+
+nRF54L-based nRF Desktop devices enable hardware cryptography for the MCUboot bootloader.
+Public key used by the MCUboot for the application image validation is securely stored in hardware Key Management Unit (KMU).
+In this use case, application image is automatically signed by the |NCS| build system, but the public key is not automatically provisioned to device together with programming the bootloader and the application images (``west flash``).
+
+Use ``west ncs-provision`` command to provision the MCUboot keys before programming the bootloader and the application images.
+Make sure that the provisioned public key is generated from the private key that was used to sign the application image.
+The private keys are stored in the application configuration directory of the board.
+Path to the used private key is defined by the ``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE`` sysbuild Kconfig option.
```

Comment:
```suggestion
Path to the private key used is defined by the ``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE`` sysbuild Kconfig option.
```
or maybe "used" isn't necessary at all if it's clear from the context?

Reply:
I can remove `used` here.